packages/account/src/Components/trading-assessment/trading-assessment-form.jsx

```diff
@@ -0,0 +1,194 @@
+import { localize, Localize } from '@deriv/translations';
+import TradingAssessmentRadioOption from './trading-assessment-radio-buttons';
+import TradingAssessmentDropdownOption from './trading-assessment-dropdown';
+
+const TradingAssessmentForm = ({
+    assessment_questions,
+    class_name,
+    form_value,
+    onSubmit,
+    onCancel,
+    is_header_navigation,
+}) => {
+    const [is_next_button_enabled, setIsNextButtonEnabled] = React.useState(false);
+    const [current_question_details, setCurrentQuestion] = React.useState({
+        current_question_index: 0,
+        current_question: {},
+    });
+    const [form_data, setFormData] = React.useState({});
+
+    const last_question_index = assessment_questions.length - 1;
+
+    React.useEffect(() => {
+        setCurrentQuestion(prevState => ({
+            ...prevState,
+            current_question: assessment_questions[prevState.current_question_index],
+        }));
+        setFormData(form_value);
+    }, []);
+
+    const displayNextPage = () => {
+        if (form_data.risk_tolerance === 'No') {
+            onSubmit(form_data, true, true);
+        } else {
+            const next_question = current_question_details.current_question_index + 1;
+            if (next_question < assessment_questions.length) {
+                setCurrentQuestion({
+                    current_question_index: next_question,
+                    current_question: assessment_questions[next_question],
+                });
+            }
+        }
+    };
+
+    const displayPreviousPage = () => {
+        const prev_question = current_question_details.current_question_index - 1;
+        if (prev_question >= 0) {
+            setCurrentQuestion({
+                current_question_index: prev_question,
+                current_question: assessment_questions[prev_question],
+            });
+        }
+    };
+
+    const handleOnClickNext = values => {
+        if (!isAssessmentCompleted(values)) {
+            displayNextPage();
+        }
+    };
+
+    const handleValueSelection = (e, form_control, callBackFn) => {
+        if (typeof e.persist === 'function') e.persist();
+        callBackFn(form_control, e.target.value);
+        setFormData(prev_form => ({ ...prev_form, [form_control]: e.target.value }));
+    };
+
+    const hideElement = condition => {
+        if (condition) {
+            return { visibility: 'hidden' };
+        }
+        return {};
+    };
+
+    const isAssessmentCompleted = answers => Object.values(answers).every(answer => Boolean(answer));
+
+    return (
+        <div className={classNames('trading-assessment', class_name)}>
+            <Text as='p' color='prominent' size='xxs' className='trading-assessment__side-note'>
+                <Localize i18n_default_text='In providing our services to you, we are required to obtain information from you in order to asses whether a given product or service is appropriate for you.' />
```

Comment:
```suggestion
                <Localize i18n_default_text='In providing our services to you, we are required to obtain information from you in order to assest whether a given product or service is appropriate for you.' />
```

Reply:
It is `assess`. Like assessment. Will change the misspelt word